.github/workflows/jdk8-release.yml

```
@@ -0,0 +1,77 @@
+name: Build and push the Docker images for JDK 8
+
+on:
+  push:
+    tags:
+      - "jdk8-[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  release-jdk8:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/jdk8-##g")
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Check out codes
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
```

```suggestion
        uses: docker/login-action@v2
```
I think we can use the latest version `v2`.
https://github.com/docker/login-action/tree/v2

---

Addressed by https://github.com/scalar-labs/docker/pull/20/commits/304580b2d7d4b0ff65d954ee5e48baaba865603e